evm-contracts/src/v0.6/dev/DeviationFlaggingValidator.sol

```diff
@@ -162,5 +155,58 @@ contract DeviationFlaggingValidator is Owned, AggregatorValidatorInterface {
     return uint256(value < 0 ? value.mul(-1): value);
   }
 
+  /**
+   * @dev Subtracts two signed integers, returns false 2nd param on overflow.
+   * Modified version of OpenZeppelin's SignedSafeMath.
+   */
+  function safeSub(int256 a, int256 b) internal pure returns (int256, bool) {
```

Comment:
Consider renaming these methods with a `silent` prefix, as the the original library functions were already safe, they just threw an Error where these ones are silently failing and returning a `bool`. 

So `silent` prefix may better describe the functions in this context.